Makefile

```diff
@@ -17,8 +17,24 @@ VENV          = $(VENVDIR)/bin/activate
 
 # Put it first so that "make" without argument is like "make help".
 help: $(VENVDIR)
```

Comment:
We can get rid of the `$(VENVDIR)` this way - no need to install the environment just to look at the help.

Reply:
> We can get rid of the `$(VENVDIR)` with this implementation - no need to install the environment just to look at the help.

Reply:
Note: resolved with https://github.com/canonical/sphinx-docs-starter-pack/pull/181/commits/258b3659b8cd97cb9923c5c7107b9297bf350c4b